packages/gasket-plugin-nextjs/test/index.test.js

```diff
@@ -274,38 +274,38 @@ describe('workbox hook', () => {
   });
 
   it('config modifies urls to use assetPrefix with https', async () => {
-    const assetPrefix = 'https://some-cdn.com/';
```

Comment:
You should retain a test to verify that `gasketAPI.config.next.assetPrefix` continue to works with this lifecycle.